### ISSUE TITLE
Transform JATS XML for content-store

### ISSUE THREAD

**Issue by giorgiosironi** [libero/libero#155]
5. Wrap JATS content with an item tag like so: 
```xml
<item xmlns="http://libero.pub" xmlns:jats="http://jats.nlm.nih.gov">
  <jats:article></jats:article>
</item>
```
6. Add `xml:base=""` attribute to `jats:artcle` tag like so:
```xml
<jats:article xml:base="https://unstable-elife-style-content-adapter-expanded.s3.amazonaws.com/<article-id>">
```

The URL should have a limited lifetime, as the files only have to be present there for the duration of the PUT request so that the `content-store` can download them. It is not meant for long-term storage of these files.

7. Add meta data to document:
```xml
<item xmlns="http://libero.pub" xmlns:jats="http://jats.nlm.nih.gov">
    <meta>
        <id>[article id]</id>
        <service>[service name from config file]</service>
    </meta>
    <jats:article></jats:article>
</item>
```

**Comment by GiancarloFusiello** [comment]
Currently blocked by https://github.com/libero/libero/issues/159

**Comment by GiancarloFusiello** [comment]
moving the validation of XML to a separate issue so it does not block this task.
